### DISCUSSION TITLE
Keys in live history mode

### DISCUSSION THREAD

**Discussion by underlow** [marlonrichert/zsh-autocomplete#487] [Q&A] [answered]
Is it possible to disable all actions keeping only Arrow Up & Down in live history mode? 

There are 2 use cases I want to cover
1. When I select some command from history and press some navigation keys (like  CtrlLeft) I want to edit selected command not select something else.  Now OptionLeft select some other command and CmdLeft and CtrlLeft simply do not work. 
2. When I type something and press Tab it selects and run highlighted command. It would be much mo convinient to switch to directory or command completion after Tab key (wouldn't break habits from other terminals) It is possible?

**Comment by marlonrichert** [comment, marked as answer]
> When I select some command from history and press some navigation keys (like CtrlLeft) I want to edit selected command not select something else. Now OptionLeft select some other command and CmdLeft and CtrlLeft simply do not work.

That's a bug and needs to be fixed: #411 

> When I type something and press Tab it selects and run highlighted command. It would be much mo convinient to switch to directory or command completion after Tab key

I'm not sure I understand. Can you give me a concrete example?
